### PR TITLE
Version 2.3.0 Release

### PR DESCRIPTION
**Pull request by itye-msft**
## Change Description

Describe your changes

## Issue reference

This PR fixes issue #XX

## Checklist

- [ ] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [ ] I have signed the CLA
- [ ] My code includes unit tests
- [ ] All unit tests and lint checks pass locally
- [ ] My PR contains documentation updates / additions if required

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,23 @@ All notable changes to this project will be documented in this file.
...
+### Changed
+* Create CODE_OF_CONDUCT
+* ADF templates docs
+* Fix spark sample to run presidio in broadcast
+* Ad-hoc recognizers
+* Text Analytics Integration Sample
+* Documentation update and samples validation
+* Adding tagger to the spaCy model pipeline
+* Sample notebook for remote recognizer (using Text Analytics)
+* Add matplotlib to image-redactor
```
Comment (SharonHart) [inline CHANGELOG.md:22]: This one is for the purpose of pii verification in the image redactor

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,23 @@ All notable changes to this project will be documented in this file.
 
 ### Deprecated
 
+## [2.3.0] - 10.05.2021
+### Changed
+* Create CODE_OF_CONDUCT
```
Comment (navalev) [inline CHANGELOG.md:14]: added custom lambda recognized to the analyzer

File: VERSION [hunk 3]
```diff
@@ -1,2 +1,2 @@
-2.2.0
+2.3.0
```
[the remark this answers is not in the document]
Comment (navalev) [reply VERSION:1]: agree. should be 2.2.1

File: CHANGELOG.md (hunk 2 above)
Comment (navalev) [reply CHANGELOG.md:14]: added add pii_verify_engine to the image-redactor

**Review by navalev — CHANGES_REQUESTED** [review]

**Review by navalev — APPROVED** [review]

**Review by SharonHart — APPROVED** [review]

**Pull request merged**